dss-api.yml

```diff
@@ -48,6 +48,12 @@ paths:
                 type: object
             required:
               - es_query
+        - name: format
+          in: query
+          description: Specifies the output format. Raw will return the json meta data in the results.
```

Comment:
You may want to quote it `` `raw` `` or do it in some way that indicates that the value should be `raw` and not `Raw`.